packages/block-editor/src/store/selectors.js

```diff
@@ -2504,6 +2504,17 @@ export function getSettings( state ) {
 	return state.settings;
 }
 
+/**
+ * Returns the behaviors.
```

Comment:
I know this is a draft, but let's expand this before merging and consider what someone will learn when they find this function. what is a behavior? when should someone be using this selector?

given that the input and output types are `Object` the types give no clues to what to expect. a simple example code snippet can go a long way

```js
/**
 * Returns behaviors registered with the editor.
 *
 * "Behaviors" mark blocks for client-side interactivity in
 * predefined ways. They can be selected for blocks which have
 * declared support through `registerBlockType` and themes
 * which have done so through `theme.json`.
 *
 * Example
 *     select( 'block-editor' ).getBehaviors() === [
 *         { name: 'lightbox', … },
 *         …
 *     ]
 *
 * @see {https://behaviors-docs/somewhere}
 */
```

I'm making up the example and descriptions because I don't personally know what should go there, but this little bit of help can save someone from manually trying to find their way through the zoo of implicit dependencies in the code.